vignettes/traduire.Rmd

```diff
@@ -61,3 +64,100 @@ translator$t("interpolate", list(what = "i18next", how = "easy"),
 translator$t("interpolate", list(what = "i18next", how = "facile"),
              language = "fr")
 ```
+
+## Use within a package
+
+We provide an optional workflow for using translations within a package, or some other piece of code where the translations will be fairly invasive to add, allowing you to write essentially:
+
+```r
+traduire::t_(...)
+```
+
+and have all the `...` arguments forwarded to the appropriate translator object. There are several details here:
+
+* how do we determine what ios the "appropriate" translator object
+* how do we determine what language is active for this translation?
+
+
+
+To do this, we allow packages (or other simiar code) to "register" a translator, like
+
+```r
+traduire::translator_register(resources)
+```
+
+where `name` is the key representing your object (more on this later) and `...` are arguments to `traduire::i18n`
```

Comment:
Is this line out of date? Where are `name` and `...` used? Similar what does `resources` on line 86 refer to?

Reply:
thanks, updated